ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -317,6 +311,44 @@ func convertToContainerDef(inputCfg *adapter.ContainerConfig, ecsContainerDef *C
 	// Set fields from ecs-params file
 	if ecsContainerDef != nil {
 		outputContDef.Essential = aws.Bool(ecsContainerDef.Essential)
+
+		overrideMsg := "Using ecs-params value as override (was %v but is now %v)"
+		if ecsContainerDef.Cpu > 0 {
+			if outputContDef.Cpu != nil && *outputContDef.Cpu > 0 {
+				log.WithFields(log.Fields{
+					"option name":  "Cpu",
+					"service name": inputCfg.Name,
+				}).Infof(overrideMsg, *outputContDef.Cpu, ecsContainerDef.Cpu)
+			}
+			outputContDef.Cpu = aws.Int64(ecsContainerDef.Cpu)
```

Comment:
Are we setting the Cpu even if it is nil here then? Relatedly, I'm wondering if it makes sense to combine the nil/empty check conditionals in line 326-327 as well as 336-337 as well, for consistency.